dcc-storage-client/src/test/java/org/icgc/dcc/storage/client/ClientMainTest.java

```diff
@@ -86,12 +86,17 @@ public void testMainDownloadWithNonExistentManifest() throws Exception {
   @Test
   public void testMainDownloadWithNonExistentOutDir() throws Exception {
     val file = tmp.newFile();
-    val outDir = new File("/foo");
+    val outDir = new File("/tmp/foo");
```

Comment:
My point is that there is a real chance that `/tmp/foo` already exists due to a failing / aborted previous test. By using the `@Rule` I suggested, this is not possible and thus is recommended.